src/core/manager.c

```diff
@@ -1868,6 +1868,13 @@ Unit *manager_get_unit(Manager *m, const char *name) {
         return hashmap_get(m->units, name);
 }
 
+const char *manager_get_unitpath(Manager *m, const char *name) {
```

Comment:
hmm, "unitpath" is not how we usually would spell that here, i.e. underscore missing at least.

hmm, given that this doesn't actually get the current unit's path but the path of what we'd load if the unit was loaded I think we should find a better verb than "get". Maybe call this "manager_lookup_unit_path()"?

Reply:
done